crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -25,6 +25,19 @@ pub struct UnsignedSoftConfirmation<'txs, Tx> {
     timestamp: u64,
 }
 
+/// TODO: doc or find workaround
```

Comment:
forgotten I assume :D

Reply:
not the TODO: but the Old blah blah stuff in general

Reply:
Cherrypicked those, lemme cherry pick the new ones 🍒

Reply:
or just revert lol